lib/aggregate.rb

```diff
@@ -23,4 +23,40 @@
 require "aggregate/container"
 
 module Aggregate
+  class << self
+    attr_writer :configuration
+
+
+    def configuration
+      @configuration ||= Configuration.new
+    end
+
+    def configure
+      yield(configuration)
+    end
+
+    def reset
+      @configuration = Configuration.new
+    end
+  end
+
+  class Configuration
+    attr_accessor :keys_list
+
+    def initialize
+      # Should be a list of keys used for decryption or a single key
+      #   - keys : to encrypt/decrypt
+      #
+      # Example:
```

Comment:
Is it a list or a hash, cause the name/comment vs. the example is ambiguous. Fwiw I think it should be a list.